react-front-end/src/components/home.js

```diff
@@ -24,13 +26,23 @@ export default function Home() {
       </div>
       <div className="featured">
         {state.games.map((game) => {
-          console.log(game);
           return <GameBox key={game.id} id={game.id} title={game.title} description={game.description} platform={game.platform} cover={game.cover} />
```

Comment:
`return <GameBox key={game.id} id={game.id} title={game.name} description={game.summary} platform={game.platforms} cover={game.cover} `

We probably need a way to drag each string out of the platforms array